### DISCUSSION TITLE
S3 bucket integration for recordings

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#624] [General]
It would be nice if the configuration allowed integration with an Amazon S3 bucket, so recorded files could be uploaded to S3 and deleted from the server automatically, while the webhook pings this information to the configured endpoint.

**Comment by jibon57** [comment]
You can easily use `[post_processing_scripts](https://github.com/mynaparrot/plugNmeet-recorder/tree/main/post_processing_scripts)` for this purpose.
